docs_guidelines/apidocs/how_to.examples.rst

```diff
@@ -0,0 +1,12 @@
+######################
+How-to Guides Examples
```

Comment:
same as above, an intro section on how to best make use of this page would be good